src/cmd/manage_nsfs.js

```diff
@@ -259,6 +266,63 @@ async function delete_bucket(data, force) {
     }
 }
 
+
+/**
+ * filter_bucket will return true or false based on whether a bucket meets the criteria defined by the supported flags.
+ * currently not implemented
+ * @param {object} bucket
+ * @param {string[]} [filters]
```

Comment:
hmm should it be an object?

Reply:
I just moved the function place.
I think your'e right, from what I tested:
I run `sudo node src/cmd/manage_nsfs account list --wide --show_secrets --gid 1001` and added logs and saw:
```
Feb-9 16:04:49.674 [/79185]   [LOG] CONSOLE:: SDSD account_filters { gid: 1001 }
Feb-9 16:04:49.674 [/79185]   [LOG] CONSOLE:: SDSD typeof account_filters object
```